components/ButtonBar/tests/unit/ButtonBarButtonTitleFontTests.swift

```diff
@@ -75,6 +75,20 @@ class ButtonBarButtonTitleFontTests: XCTestCase {
     }
   }
 
+  func testCustomFontChangesButtonFrames() {
```

Comment:
Please make sure there are tests for both the `item.width > 0` and `MDCCGFloatEqual(item.width, 0)` case.  I'm curious how the branching code above is expected to work.

Reply:
Done.